db/migrate/20250217092158_add_export_info_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddExportInfoToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :exportable_columns, :jsonb, default: {}
```

Comment:
```suggestion
    add_column :users, :export_settings, :jsonb, default: {}
```
It is no more limited to the columns, so I would prefer to have a more generic naming